plugins/importexport/users/PKPUserImportExportPlugin.inc.php

```diff
@@ -213,7 +216,8 @@ function importUsers($importXml, $context, $user) {
 		$importFilter = array_shift($userImportFilters);
 		$importFilter->setDeployment(new PKPUserImportExportDeployment($context, $user));
 
-		return $importFilter->execute($importXml);
+		$importFilter->execute($importXml);
+		return $importFilter;
```

Comment:
This change would require a change to the `@return` self-documentation. But rather than what you've done here, I'd suggest adding the filter as a reference parameter to the `importUsers` function, and leaving the return convention here as it used to be (which makes more sense to me).